src/controllers/customercontroller.js

```diff
@@ -6,29 +6,81 @@ class CustomerController extends OrderController
             "src/views/template/customer.html",
             "#customers",
             {}
-        );
+        ).done(() => this.registerEvent());
+    }
+
+    registerEvent()
+    {
+        $(document).on('click', '.customer-complete', function(e) {
```

Comment:
Not working on elements created after the registerEvent method has been called. That's why I went with this version.